tests/testlib.nix

```diff
@@ -1,5 +1,26 @@
 { lib }:
-{
+
```

Comment:
This sounds good, I'd like if we could merge that together with the separate PR from the first change in here.